### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by sio4** [gobuffalo/buffalo#2307] [Show and tell]
## Congratulations :tada: Buffalo core v1!

(No code changes since v0.18.9, just a few module dependency updates)

**Full Changelog**: https://github.com/gobuffalo/buffalo/compare/v0.18.9...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/gobuffalo/buffalo/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by bzins3r** [comment]
Congrats!!!!!

**Comment by rnice01** [comment]
Congrats!!!! I can't imagine what a journey it's been for you all, but I'm so excited for you all!

**Comment by hubyhuby** [comment]
Bravo! I have been following this community and tickering with buffalo for some years now. Great team very kind and hard working. What a release!

**Reply by hubyhuby** [reply]
It has been a long road , Mark starting with a new language line go and developping a framework. A lot of movement in go itself made a lot a movement in buffalo.
Now the new team pushed it to the last miles to 1.0. it wasn t granted that the comminity of developper would get there.
Big achievement. I am also impressed how different devloppers from différent horizon improve the code from various angles.
